R/eye_msg_report.R

```diff
@@ -0,0 +1,52 @@
+
+eye_msg_report <- function(eye, report_path = NULL, events = "all", return_eye = FALSE){
+  #for a quick look at ET message flow by event. Allows for user to quickly look for messaging conventions that can be specified in YAML config file.
+
+  if(is.character(eye)){ # if string provided, attempt to read it.
+    eye <- read_edf(file, keep_asc=FALSE, parse_all=TRUE, samples = FALSE)[[1]] #samples = FALSE removes read-in of raw data
+  }
+
+  stopifnot(exists("eye")) # must be input as argument or an edf must supply it. Can be read internally, supplied as the output of read_edf, or as an already initialized ep.eye object.
```

Comment:
This should precede if (is.character()) since the former will error if eye is missing